Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -72,15 +88,21 @@ def __init__( self, ceUniqueID ):
     self.outputURL = 'gsiftp://localhost'
     self.gridEnv = ''
     self.proxyRenewal = 0
+    self.extraSubmitString = self.ceParameters.get('ExtraSubmitString', '').decode('string_escape')
+
+    ## see note on getCondorLogFile, why we can only use the global setting
+    self.workingDirectory = gConfig.getValue( "Resources/Computing/HTCondorCE/WorkingDirectory",
```

Comment:
I tried instantiating a HTCondorCEComputingElement to get the log file, but the ceParameter locations at the Site and CE level are only resolved in the getQueues function that is called via the SiteDirector, so it doesn't work that way.